vector/src/main/java/im/vector/app/features/navigation/DefaultNavigator.kt

```diff
@@ -176,6 +176,9 @@ class DefaultNavigator @Inject constructor(
             Navigator.PostSwitchSpaceAction.OpenAddExistingRooms -> {
                 startActivity(context, SpaceManageActivity.newIntent(context, spaceId, ManageType.AddRooms), false)
             }
+            Navigator.PostSwitchSpaceAction.OpenRoomList -> {
+                startActivity(context, SpaceExploreActivity.newIntent(context, spaceId), false)
```

Comment:
would be handy to have a named parameter for the `false` argument

Reply:
I agree it's better to use named argument for Boolean parameters, as a global rule 💯